package.json

```diff
@@ -65,5 +65,6 @@
   },
   "resolutions": {
     "natives": "1.1.6"
-  }
-}
+  },
+  "main": "build/electron/electron/index.js"
```

Comment:
I find it odd to have this at the root of the repo. Can we put in the electron directory or pass as a build command line flag?

Reply:
I don't know of a good alternative. `electron-builder` and `electron` have options to override the *root folder* but not the main *class* which has to be at a certain location within that root folder (we *could* add a one-liner at `/index.js` - we currently do that and I was happy to eliminate it in this PR).

I think switching to workspaces is the real solution here but I definitely don't want to tackle that right now.